pym/bob/input.py

```diff
@@ -2982,6 +2982,7 @@ def __init__(self):
         self.__validTypes = schema.Schema({'backend': schema.Or('none', 'file', 'http', 'shell', 'azure')},
             ignore_extra_keys=True)
         baseArchive = {
+            'name' : str,
```

Comment:
The `name` attribute must be optional. Otherwise we'll break a lot of existing projects. That will also save you the test adaptions in large part I guess.